### DISCUSSION TITLE
How to display components that uses store created using zustand in Storybook ?

### DISCUSSION THREAD

**Discussion by asvny** [pmndrs/zustand#310] [Q&A]
The more idiomatic way is presentational/container components but wondering if there is any other way.

Take for example Todo list .. it can be in different states and I would like to have stories for each of them like
1. yet to be done
2. partially completed
3. all completed
4. Editing stating

In the case of redux, I can wrap it with a provider and give an appropriate initial state .. so automatically I can force it to be in my desired state.

Coming to zustand .. I like the simplicity and but how do I give different initial states for each story ?

**Comment by dai-shi** [comment]
This probably falls into how to use zustand with provider.
We have a long discussion in #182, which is a bit stalled.
